### PR TITLE
Add youtube and chat links for coord meeting

### LINKED ISSUE #315: Add link to YouTube stream for 2022 coordination meeting

### LINKED ISSUE #316: Add link to coordination meeting chat to 2022 meeting page
https://matrix.to/#/#sunpycoordinationmeeting:openastronomy.org

### PR DESCRIPTION
**Pull request by Cadair**
Fixes #315 fixes #316

### CODE REVIEW

**Review by hayesla — APPROVED** [review]

File: project/meetings.rst [hunk 1]
```diff
@@ -21,6 +21,10 @@ The link to join the meeting as well as a link to the meeting agenda are include
 
 * `Link to join the meeting <https://sunpy.org/jitsi>`__
 
+* `YouTube Livestreams and recordings for each day <https://www.youtube.com/playlist?list=PLOWSPnooGuj2sWk3enDieOE_H-WvcZZCd>`__
+
+* `Chat room for the meeting <https://openastronomy.element.io/#/room/#sunpycoordinationmeeting:openastronomy.org>`__
+
```
Comment (wtbarnes) [inline project/meetings.rst:27]: These should go under the 2022 meeting heading, not the weekly meeting heading

Comment (Cadair) [reply project/meetings.rst:27]: Baaah

Comment (nabobalis) [reply project/meetings.rst]: Moved.

File: project/meetings.rst [hunk 2]
```diff
@@ -774,7 +774,11 @@ to accommodate participants in the U.S.
 Virtual Participation
 ---------------------
 
-Details regarding virtual participation will be posted here closer to the time of the meeting.
+* `Coordination Video Meeting Room <https://sunpy.org/jitsi>`__
+
+* `YouTube Livestreams and recordings for each day <https://www.youtube.com/playlist?list=PLOWSPnooGuj2sWk3enDieOE_H-WvcZZCd>`__
+
+* `Chat room for the meeting <https://openastronomy.element.io/#/room/#sunpycoordinationmeeting:openastronomy.org>`__
```
Comment (Cadair) [inline project/meetings.rst:781]: The chat room should probably not just be under the virtual heading?

Comment (nabobalis) [reply project/meetings.rst:781]: K

Comment (nabobalis) [reply project/meetings.rst]: Moved

**Pull request merged**